build.gradle.kts

```diff
@@ -128,6 +128,8 @@ extra["versions.jsr305"] = "1.3.9"
 extra["versions.jansi"] = "1.16"
 extra["versions.jline"] = "3.3.1"
 extra["versions.junit"] = "4.12"
+extra["versions.junit5"] = "5.2.0"
+extra["versions.junit-platform"] = "1.2.0"
```

Comment:
Here we specify the versions that will be used throughout the entire project,  e.g. in case if Kotlin project would be using JUnit 5 to run its own tests. That version should not necessary be the same as the version against which kotlin-test-junit5 is being built.

Reply:
So, better to move concrete version to library project? 
I have just take the latest version of junit, and forgot to remove `versions.junit-platform`(it's not longer needed because gradle 4.6 now has native support of junit5)